src/Native/Runtime/unix/PalRedhawkUnix.cpp

```diff
@@ -334,7 +334,7 @@ int UTF8ToWideChar(const char* bytes, int len, wchar_t* buffer, int bufLen)
     size_t inbufbytesleft = len;
     size_t outbufbytesleft = bufLen;
 
-    int rc = iconv(cd, &inbuf, &inbufbytesleft, &outbuf, &outbufbytesleft);
+    int rc = iconv(cd, const_cast<const char **>(&inbuf), &inbufbytesleft, &outbuf, &outbufbytesleft);
```

Comment:
It seems that it broke on MacOSX and Linux.. while it worked 

```
16:07:05     int rc = iconv(cd, const_cast<const char **>(&inbuf), &inbufbytesleft, &outbuf, &outbufbytesleft);
16:07:05              ^~~~~
16:07:05 /usr/include/iconv.h:69:8: note: candidate function not viable: no known conversion from 'const char **' to 'char **' for 2nd argument
16:07:05 size_t iconv (iconv_t /*cd*/,
16:07:05        ^
```

How would you address this?